DeliveryCompany.BusinessLayer/PackageService.cs

```diff
@@ -94,16 +138,36 @@ public List<Package> GetPackagesWithStatus(StateOfPackage stateOfPackage)
             }
         }
 
+        public List<Package> GetPackagesWithStatusOnAutomaticWaybill(StateOfPackage stateOfPackage)
+        {
+            using (var context = _deliveryCompanyDbContextFactoryMethod())
+            {
+                return context.Packages
+                    .Include(x => x.Sender)
+                    .Where(x => (x.State == stateOfPackage && x.ModeWaybill==ModeOfWaybill.Automatic && (x.RecipientLat != 999 || x.RecipientLon != 999)))
+                    .ToList();
+            }
+        }
+
+        public List<Package> GetPackagesWithStatusOnManualWaybill(StateOfPackage stateOfPackage)
```

Comment:
As previously, GetPackagesWithStatusOnManualWaybill and GetPackagesWithStatusOnAutomaticWaybill method can be one, just additional param needs to be provided.